app/controllers/application_controller.rb

```diff
@@ -127,12 +128,12 @@ def after_sign_in_path_for(resource)
     stored_location_for(resource) || user_path(resource)
   end
 
-  def after_sign_out_path_for(resource)
+  def after_sign_out_path_for(_resource)
```

Comment:
This might actually be unused now... I think it was something for the old Devise login pages